QSB/QSB.cs

```diff
@@ -32,5 +26,11 @@ private void Start()
             gameObject.AddComponent<PreserveTimeScale>();
         }
 
+        private void Update()
+        {
+            Cursor.lockState = CursorLockMode.None;
+            Cursor.visible = true;
```

Comment:
Actually we should get rid of this stuff with the cursor. It was only useful back when you had to connect after waking up.